projects/valtimo/form-management/src/lib/form-management-edit/form-management-edit.component.ts

```diff
@@ -256,7 +256,9 @@ export class FormManagementEditComponent
       language: 'json',
     });
 
-    this.reloading$.next(false);
+    setTimeout(() => {
+      this.reloading$.next(false);
+    }, 100);
```

Comment:
does it also work without a value or with 0?

Reply:
No, only with a value of 100 or more